packages/mds-stream/kafka/agency-stream-kafka.ts

```diff
@@ -15,15 +15,17 @@
  */
 
 import { VehicleEvent, Telemetry, Device, TripMetadata } from '@mds-core/mds-types'
-import { ClientDisconnectedError, getEnvVar } from '@mds-core/mds-utils'
+import { getEnvVar } from '@mds-core/mds-utils'
 import { KafkaStreamProducer } from './stream-producer'
 import { AgencyStreamInterface } from '../agency-stream-interface'
+import { safeWrite } from '../helpers'
 
 const { TENANT_ID } = getEnvVar({
   TENANT_ID: 'mds'
 })
 const deviceProducer = KafkaStreamProducer<Device>(`${TENANT_ID}.device`)
 const eventProducer = KafkaStreamProducer<VehicleEvent>(`${TENANT_ID}.event`)
+const eventErrorProducer = KafkaStreamProducer<Partial<VehicleEvent>>(`${TENANT_ID}.event-error`)
```

Comment:
@avatarneil should this be snake instead of BBQ-case?

Reply:
For consistency naming may want to look something like `event.error` following the pattern we use for other topics like `event.annotated`.

Reply:
fixed!